ajax/libs/academicons/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "academicons",
+  "filename": "css/academicons.min.css",
+  "description": "Academicons is an icon font for academics. It aims to supplement other packages (such as Font Awesome) with icons for academic websites.",
+  "homepage": "https://jpswalsh.github.io/academicons/",
+  "keywords": [
+    "css",
+    "font",
+    "icons",
+    "academic",
+    "research",
+    "university"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jpswalsh/academicons.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/academicons*",
+          "fonts/academicons*"
+        ]
+      }
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jpswalsh/academicons.git"
+  },
+  "licenses": [
+    "OFL-1.1",
+    "MIT"
+  ],
+  "author": {
+    "name": "James Walsh",
+    "email": "james.walsh@northwestern.edu",
+    "web": "http://jpswalsh.com/"
```

Comment:
Let's use `"url"` instead of `"web"`. Follow the package.json format on NPM. ref: https://docs.npmjs.com/files/package.json#people-fields-author-contributors. Thanks.